components/collection/drop/modal/paid/MintOverview.vue

```diff
@@ -26,7 +26,7 @@
               :src="
                 toMintNft.imageDataPayload
                   ? toMintNft.imageDataPayload.image
-                  : sanitizeIpfsUrl(toMintNft.image)
+                  : sanitizeIpfsUrl(toMintNft.animationUrl)
```

Comment:
so are youu still using full drop right?

Reply:
This is needed to load once the iframe and capture the image data payload , after that only images are used